sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -193,6 +227,12 @@ internal class RadarApiClient(
             params.putOpt("locationAuthorization", RadarUtils.getLocationAuthorization(context))
             params.putOpt("locationAccuracyAuthorization", RadarUtils.getLocationAccuracyAuthorization(context))
             params.putOpt("sessionId", RadarSettings.getSessionId(context))
+            params.putOpt("trackingOptions", RadarSettings.getTrackingOptions(context).toJson())
+
+            val listenToServer = RadarSettings.getListenToServerTrackingOptions(context)
+            if (listenToServer) {
+                params.putOpt("listenToServer", listenToServer)
+            }
```

Comment:
The docs for `putOpt` specify that it does nothing if either value is null, so you should be able to simplify it to:

```suggestion
            params.putOpt("listenToServer", RadarSettings.getListenToServerTrackingOptions(context))
```

Reply:
`listenToServer` is of type `Boolean` instead of `Boolean?`, so I figured it'd save about a couple bytes not to send it up.